src/app/core/learning-object-module/learning-object/learning-object.routes.ts

```diff
@@ -68,13 +68,10 @@ export const LEGACY_COLLECTIONS_ROUTES = {
      * Request to update the collection of a learning object
      * @method PATCH
      * @auth required
-     * @param username - The username of the author of the learning object
      * @param learningObjectCuid - The cuid of the learning object to update
      */
-    UPDATE_LEARNING_OBJECT_COLLECTION(username: string, learningObjectCuid: string) {
-        return `${environment.apiURL}/users/${encodeURIComponent(
-            username
-        )}/learning-objects/${encodeURIComponent(
+    UPDATE_LEARNING_OBJECT_COLLECTION(learningObjectCuid: string) {
```

Comment:
should this be moved over to `LEARNING_OBJECT_ROUTES` or does it stay here?

Reply:
i believe this needs to be moved but check w @mfranz1

Reply:
I agree that this should be moved.

Reply:
+1, please move to LEARNING_OBJECT_ROUTES

Reply:
The route needs to be updated to `collections`